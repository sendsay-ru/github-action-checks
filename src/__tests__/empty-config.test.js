const { render } = require('../utils');
const source = require('../../config/source');
const emptyConfig = require('../../config/empty');

describe('Render by empty config', () => {
  it('should expose a function', () => {
    expect(render).toBeDefined();
  });

  it('should return expected value', () => {
    expect(render(source, emptyConfig)).toMatchSnapshot();
  });
});
