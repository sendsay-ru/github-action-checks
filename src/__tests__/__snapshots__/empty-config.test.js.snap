// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render by empty config should return expected value 1`] = `[]`;
